pkg/roachpb/api.proto

```diff
@@ -2793,14 +2795,17 @@ message RangeFeedSSTable {
   bytes              data     = 1;
   Span               span     = 2 [(gogoproto.nullable) = false];
   util.hlc.Timestamp write_ts = 3 [(gogoproto.nullable) = false, (gogoproto.customname) = "WriteTS"];
+  Span               registered_span     = 4 [(gogoproto.nullable) = false];
```

Comment:
This seems a bit iffy. Shouldn't the client know this and keep track of it? Is it because the DistSender hides this internally?

Also, I see why we might need this on the SST (which isn't trimmed), but the DeleteRange should be truncated to the registration bounds already, so I don't think it should be necessary there in any case?